### PR TITLE
1.30.1 update

### PR DESCRIPTION
**Pull request by Vardominator**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Current node version: **1.29.0**
 
 `TYPE=[pool|relay]`
 
-`VERSION=1.29.0`
+`VERSION=1.30.1`
```
Comment (ggydush) [inline README.md:14]: What is this section again?

File: scripts/adjust_margin.sh [hunk 2]
```diff
@@ -80,14 +80,15 @@ cardano-cli transaction build-raw \
...
+# fee=$(cardano-cli transaction calculate-min-fee \
+#     --tx-body-file tx.tmp \
+#     --tx-in-count ${txcnt} \
+#     --tx-out-count 1 \
+#     --mainnet \
+#     --witness-count 3 \
+#     --byron-witness-count 0 \
+#     --protocol-params-file params.json | awk '{ print $1 }')
+fee=212933
 echo fee: $fee
```
Comment (ggydush) [inline scripts/adjust_margin.sh:92]: Did we ever figure out what was up with this?

**Review by ggydush — APPROVED** [review]
Wooo, thanks for the quick turnaround on this! 🎉

File: README.md (hunk 1 above)
Comment (Vardominator) [reply README.md:14]: See `${VERSION}` below

File: scripts/adjust_margin.sh (hunk 2 above)
Comment (Vardominator) [reply scripts/adjust_margin.sh:92]: nope

**Pull request merged**